packages/core/src/api/firmware/firmwareBase.ts

```
@@ -0,0 +1,378 @@
+        // @ts-expect-error
+        if (bootRes.type === 'CallMethodError') {
+          throw ERRORS.TypedError(HardwareErrorCode.FirmwareUpdateAutoEnterBootFailure);
+        }
+        this.postTipMessage(FirmwareUpdateTipMessage.GoToBootloaderSuccess);
+        this.checkDeviceToBootloader(this.payload.connectId);
+
+        // force clean classic device cache so that the device can initialize again
+        if (DeviceModelToTypes.model_classic.includes(deviceType)) {
+          DevicePool.clearDeviceCache(uuid);
+        }
+        delete DevicePool.devicesCache[''];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)